modules/client-configuration/pages/non-suse-clients.adoc

```diff
@@ -0,0 +1,6 @@
+[[non-suse-clients]]
+= Other Clients
+
+It is possible to register clients using operating systems from Red Hat, CentOS, or Ubuntu.
+
+This section contains information specific to clients running operating systems other than those provided by {suse}.
```

Comment:
Wondering whether this also applies for opensuse?

Reply:
I would include opensuse as a suse OS.